FBTweak/FBTweakCategory.m

```diff
@@ -70,4 +74,18 @@ - (void)removeTweakCollection:(FBTweakCollection *)tweakCollection
   [_namedCollections removeObjectForKey:tweakCollection.name];
 }
 
+- (void)reset {
+  for (FBTweakCollection *collection in self.tweakCollections) {
+    for (FBTweak *tweak in collection.tweaks) {
+      if (!tweak.isAction) {
```

Comment:
Remind me why what this block does? I would expect that `teak` will be reset to a default value?

Reply:
RIght now, the current value and the default value are totally separate. This means that to get the actual value of a tweak you need to `tweak.currentValue ?: tweak.defaultValue`.
In one of the next PRs I make this a bit more complex but more intuitive by adding a getter to `currentValue` that returns the default value if the current value is nil.

This is weird since when you do `tweak.currentValue = nil` and then get `currentValue`, it won't be `nil`.

Plus I add a `reset` method to tweaks, so here I'll just call `[tweak reset]` which make more sense.